spec/lib/twingly/http_spec.rb

```diff
@@ -468,6 +476,21 @@ class CustomError < StandardError; end
     end
   end
 
+  describe "#initialize" do
+    let(:client) { described_class.new(base_user_agent: base_user_agent) }
+
+    context "when no logger is given" do
+      it "defaults to a Logger" do
+        expect(client.logger).to be_a(Logger)
+      end
+
+      it "defaults to 'dev/null'" do
```

Comment:
An alternative way to test this is to actually log something and check that no output is given. I'm open to that change in case we want to avoid using `instance_variable_get ` :)

Reply:
> An alternative way to test this is to actually log something and check that no output is given. I'm open to that change in case we want to avoid using `instance_variable_get ` :)

I think what you suggest is what we need, if `Logger` would change its internals (like regarding the naming of the variable for instance) we would end up with strange errors here

Reply:
Changed in 93d9393